episodes/wrap-up3.md

```diff
@@ -20,10 +20,66 @@ After completing this episode, participants should be able to...
 
 ::::::::::::::::::::::::::::::::::::::::::::::::::
 
+We have reached the end of this training.
+
+## Looking Back
+
+Throughout this training we promoted a collaborative approach to lesson development and maintenance (as many other initiatives in The Carpentries community are conducted). Collaboration helps bring different skills and perspectives to a project and share the load of the work required to bring the project successfully past the finishing line. For lesson projects, this shared effort also extends to maintaining and updating the lesson after the initial development phase.
+
+::::::::::::: challenge
+
+# Exercise: What steps can you take to be a good collaborator? (20 minutes)
+
+Spend some time exploring at least two of the following lesson repositories:
+
+- [Collaborative Lesson Development Training](https://github.com/carpentries/lesson-development-training) (this lesson)
+- [Library Carpentry: Introduction to Working with Data (Regular Expressions)](https://github.com/librarycarpentry/lc-data-intro/) (official Library Carpentry lesson)
+- [Introduction to Geospatial Raster and Vector Data with Python](https://github.com/carpentries-incubator/geospatial-python) (in Carpentries Incubator)
+- [Introduction to R for Metagenomics](https://github.com/carpentries-lab/metagenomics-R) (in Carpentries Lab)
+
+and noting down what things you noticed in these repositories that could aid collaboration. Trainers will then lead a discussion on what people found and steps you could take to improve your lesson repository. 
+
+::: solution 
```

Comment:
I think this block should be in the main text as it is a very important list and I think it would be good not to hide it in the solution box.  Maybe a case for making the challenge a discussion as well and then having the ten tips come right after as their own section with a header.

Reply:
That's probably a good idea.

Reply:
+1

Reply:
Fixed via 35e6912a.